VAMobile/src/screens/BenefitsScreen/ClaimsScreen/NoClaimsAndAppealsAccess/NoClaimsAndAppealsAccess.test.tsx

```diff
@@ -1,39 +1,19 @@
 import 'react-native'
 import React from 'react'
-// Note: test renderer must be required after react-native.
 
-import 'jest-styled-components'
-import { ReactTestInstance, act } from 'react-test-renderer'
-
-import { context, render, RenderAPI, waitFor } from 'testUtils'
+import { screen } from '@testing-library/react-native'
+import { context, render } from 'testUtils'
 import NoClaimsAndAppealsAccess from './NoClaimsAndAppealsAccess'
-import { ClickToCallPhoneNumber, TextView } from 'components'
 
 context('NoClaimsAndAppealsAccess', () => {
-  let component: RenderAPI
-  let testInstance: ReactTestInstance
-
   beforeEach(() => {
-    component = render(<NoClaimsAndAppealsAccess />)
-
-    testInstance = component.UNSAFE_root
+    render(<NoClaimsAndAppealsAccess />)
   })
 
   it('initializes correctly', async () => {
-    expect(component).toBeTruthy()
-  })
-
-  it('should render the title', async () => {
-    expect(testInstance.findAllByType(TextView)[0].props.children).toEqual("We can't find any claims information for you")
-  })
-
-  it('should render the body', async () => {
-    expect(testInstance.findAllByType(TextView)[1].props.children).toEqual(
-      "We're sorry. We can't find any claims for you in our records. If you think this is an error, call the VA benefits hotline.",
-    )
-  })
-
-  it('should render ClickToCallPhoneNumber component', async () => {
-    expect(testInstance.findAllByType(ClickToCallPhoneNumber).length).toEqual(1)
+    expect(screen.getByText("We can't find any claims information for you")).toBeTruthy()
+    expect(screen.getByText("We're sorry. We can't find any claims for you in our records. If you think this is an error, call the VA benefits hotline.")).toBeTruthy()
+    expect(screen.getByText('800-827-1000')).toBeTruthy()
+    expect(screen.getByText('TTY: 711')).toBeTruthy()
```

Comment:
Can use `getByRole` here also
```suggestion
    expect(screen.getByRole('link', { name: '800-827-1000' })).toBeTruthy()
    expect(screen.getByRole('link', { name: 'TTY: 711' })).toBeTruthy()
```

Reply:
updated